### DISCUSSION TITLE
Is this repo only for GSSOC ?

### DISCUSSION THREAD

**Discussion by NanoTeddy** [VaibhavArora314/StyleShare#536] [Q&A] [answered]
I am browsing through out the github and this repo caught my eye.
So, my concern is that after GSSOC will this repo be active for other contribution ?

**Comment by MeetDOD** [comment, marked as answer]
Ya bro this is for open source the admin is very active you can work on this whenever you want without any restriction 👍 
This is open source project under GSSOC and will be available after GSSOC also no worries !

**Reply by VaibhavArora314** [reply]
Exactly what I wanted to say.
